app/Http/Controllers/StatusController.php

```diff
@@ -299,7 +301,8 @@ public static function ExportStatuses($startDate,
                                       'Kilometer',
                                       'Punkte',
                                       'Status',
-                                      'Zwischenhalte'
+                                      'Zwischenhalte',
+                                      'Fahrttyp'
```

Comment:
Translation?

Reply:
So here's one big problem: The csv never has been translated and I'm not sure if it's a good idea to change that __now__, because this would cause breaking changes for some users? Idk.

Reply:
Okay, but then it would be nice if we create an issue for this.

Reply:
#249